### PR TITLE
Added remindme command

### PR DESCRIPTION
**Pull request by jorenvandeweyer**
syntax: `++remindme <time> "message to remind you about"`

- When there's no valid time found the default time will be set to 1 day.
- Message is optional

formats
reminder

### CODE REVIEW

File: src/main/kotlin/me/aberrantfox/hotbot/commandframework/commands/UtilityCommands.kt [hunk 1]
```diff
@@ -107,6 +107,14 @@ fun utilCommands() = commands {
             it.respond("${target.fullName()}'s account was made on ${target.creationTime}")
         }
     }
+
+    command("remindme") {
+        expect(ArgumentType.Sentence)
```
Comment (AberrantFox) [inline src/main/kotlin/me/aberrantfox/hotbot/commandframework/commands/UtilityCommands.kt]: This command should expect a word and then a sentence, you should then use the word as the time string.

**Comment by AberrantFox** [comment]
Discussed with you in private, we'll keep the parsing format + input types. One question, does it need the quotes? I'd rather it not need those.

**Comment by jorenvandeweyer** [comment]
The quotes prevents time references in the title from being parsed.
Besides that, unresolved time references would stay in the title.

**Comment by AberrantFox** [comment]
The correct way to do that would be via a splitter, since that's the built in way to have multiple sentence like args. However, I'm pretty sure the parser can change to accommodate for no quote or splitter, just split string into a list of words, then `takeWhile` it is clearly a time string element. Then you have both strings and you can have time string stuff in the title, too.

**Comment by AberrantFox** [comment]
Rejected as implemented as per my spec in https://github.com/AberrantFox/hotbot/commit/6db042e5c718fc6b3175ce9bb0d3fce8bcad59fa

**Pull request closed without merging**